tests/atomizer.js

```diff
@@ -307,6 +307,53 @@ describe('getConfig()', function () {
         config = atomizer.getConfig(classNames, defaultConfig);
         expect(config).to.deep.equal(atomizer.mergeConfigs([defaultConfig, expectedConfig]));
     });
+
+    it("should append the default unit", function () {
+        var classNames = ['P-1', 'M-0.3', 'P-2%', 'C-333'];
+
+        var defaultConfig = {
+            defaultUnit: 'px',
+            margin: {
+                defaultUnit: 'em'
```

Comment:
@3den I don't really agree about the "finance migration" though. The goal is to *unify* our styling to allow sharing components (increase portability). So even if I see value in this change for devs *outside* of Yahoo (who are not concerned by portability and such), I do not think it should be used *inside* Yahoo unless it is set at the top-level config. In other words, if we were settings a *default* unit, it would be the *same for everybody*.